gateway/src/main/java/io/zeebe/gateway/impl/job/LongPollingActivateJobsRequest.java

```diff
@@ -85,7 +85,7 @@ public void onResponse(ActivateJobsResponse grpcResponse) {
       try {
         responseObserver.onNext(grpcResponse);
       } catch (Exception e) {
-        LOG.warn("Failed to send response {}", grpcResponse, e);
+        LOG.warn("Failed to send response to client.", grpcResponse, e);
```

Comment:
please remove also the response object from the parameter list

Reply:
Oh yes :see_no_evil: done